test/core.test.js

```diff
@@ -226,6 +226,21 @@ describe('core', () => {
     assert.equal((await p).toString(), 'foo\n')
   })
 
+  test('ProcessPromise: implements json()', async () => {
```

Comment:
Not `ProcessPromise` here but `ProcessOutput`.

Reply:
same for `text()` and `buffer()`? I just followed `toString()` test.

Reply:
Same for `valueOf()` also.

Reply:
@antongolub should I also change `ProcessOutput` for `toString()` test?